code/src/java/pcgen/core/display/CharacterDisplay.java

```diff
@@ -1045,7 +1045,7 @@ public int getECL()
 
 	public PCLevelInfo getLevelInfo(int index)
 	{
-		return levelInfoFacet.get(id, index);
+		return levelInfoFacet.get(id, index).get();
```

Comment:
This makes no sense.  Some callers of getLevelInfo are gating the result on null, why does the get appear in this method?